bitchatShareExtension/ShareViewController.swift

```diff
@@ -35,8 +35,9 @@ final class ShareViewController: UIViewController {
             statusLabel.leadingAnchor.constraint(greaterThanOrEqualTo: view.layoutMarginsGuide.leadingAnchor),
             statusLabel.trailingAnchor.constraint(lessThanOrEqualTo: view.layoutMarginsGuide.trailingAnchor)
         ])
-
-        processShare()
+        DispatchQueue.global(qos: .default).async {
```

Comment:
isn't `qos: .default` is the default option? maybe we could remove it? and just have `.global.async`?